Build error on MacOS: incompatible integer to pointer conversion  (RELAPACK_zgemmt)

I am attempting to install Petsc on MacOS, which involves downloading Openblas. However, the build of Openblas failed with the following error
<!--
```
make CC=/Users/zzyang/workspace/repos/petsc/arch-test-openblas/bin/mpicc FC=/Users/zzyang/workspace/repos/petsc/arch-test-openblas/bin/mpif90  TARGET=GENERIC USE_THREAD=0 USE_LOCKING=1 USE_OPENMP=0 MAKE_NB_JOBS=10  USE_OPENMP=0  USE_THREAD=0  NO_EXPRECISION=1  libs netlib re_lapack shared 
```
-->

```
src/lapack_wrappers.c:570:81: error: incompatible integer to pointer conversion passing 'blasint' (aka 'int') to parameter of type 'const blasint *' (aka 'const int *'); take the address with & [-Wint-conversion]
    RELAPACK_sgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, info);
                                                                                ^~~~
                                                                                &
src/../inc/relapack.h:74:216: note: passing argument to parameter here
void RELAPACK_sgemmt(const char *, const char *, const char *, const blasint *, const blasint *, const float *, const float *, const blasint *, const float *, const blasint *, const float *, float *, const blasint *);
                                                                                                                                                                                                                       ^
src/lapack_wrappers.c:583:81: error: incompatible integer to pointer conversion passing 'blasint' (aka 'int') to parameter of type 'const blasint *' (aka 'const int *'); take the address with & [-Wint-conversion]
    RELAPACK_dgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, info);
                                                                                ^~~~
                                                                                &
src/../inc/relapack.h:75:221: note: passing argument to parameter here
void RELAPACK_dgemmt(const char *, const char *, const char *, const blasint *, const blasint *, const double *, const double *, const blasint *, const double *, const blasint *, const double *, double *, const blasint *);
                                                                                                                                                                                                                            ^
src/lapack_wrappers.c:596:81: error: incompatible integer to pointer conversion passing 'blasint' (aka 'int') to parameter of type 'const blasint *' (aka 'const int *'); take the address with & [-Wint-conversion]
    RELAPACK_cgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, info);
                                                                                ^~~~
                                                                                &
src/../inc/relapack.h:76:216: note: passing argument to parameter here
void RELAPACK_cgemmt(const char *, const char *, const char *, const blasint *, const blasint *, const float *, const float *, const blasint *, const float *, const blasint *, const float *, float *, const blasint *);
                                                                                                                                                                                                                       ^
src/lapack_wrappers.c:609:81: error: incompatible integer to pointer conversion passing 'blasint' (aka 'int') to parameter of type 'const blasint *' (aka 'const int *'); take the address with & [-Wint-conversion]
    RELAPACK_zgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, info);
                                                                                ^~~~
                                                                                &
src/../inc/relapack.h:77:221: note: passing argument to parameter here
void RELAPACK_zgemmt(const char *, const char *, const char *, const blasint *, const blasint *, const double *, const double *, const blasint *, const double *, const blasint *, const double *, double *, const blasint *);
                                                                                                                                                                                                                            ^
4 errors generated.
```

 In the code, as shown below, an integer variable named 'info' is created and passed into the function. Is this necessary? Should it be 'ldc' instead?
https://github.com/OpenMathLib/OpenBLAS/blob/79cb121ab91285c7d655adc03ee33e6edbecc5fd/relapack/src/lapack_wrappers.c#L600-L611

--- a/relapack/src/lapack_wrappers.c
+++ b/relapack/src/lapack_wrappers.c
@@ -566,8 +566,7 @@ void LAPACK(sgemmt)(
     const float *B, const blasint *ldB,
     const float *beta, float *C, const blasint *ldC
 ) {
-	blasint info;
-    RELAPACK_sgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, info);
+    RELAPACK_sgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, ldC);
 }
 #endif
 
@@ -579,8 +578,7 @@ void LAPACK(dgemmt)(
     const double *B, const blasint *ldB,
     const double *beta, double *C, const blasint *ldC
 ) {
-	blasint info;
-    RELAPACK_dgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, info);
+    RELAPACK_dgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, ldC);
 }
 #endif
 
@@ -592,8 +590,7 @@ void LAPACK(cgemmt)(
     const float *B, const blasint *ldB,
     const float *beta, float *C, const blasint *ldC
 ) {
-	blasint info;
-    RELAPACK_cgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, info);
+    RELAPACK_cgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, ldC);
 }
 #endif
 
@@ -605,7 +602,6 @@ void LAPACK(zgemmt)(
     const double *B, const blasint *ldB,
     const double *beta, double *C, const blasint *ldC
 ) {
-	blasint info;
-    RELAPACK_zgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, info);
+    RELAPACK_zgemmt(uplo, transA, transB, n, k, alpha, A, ldA, B, ldB, beta, C, ldC);
 }
 #endif

